### DISCUSSION TITLE
Is it possible to support `.cookie()` with `.customize()`?

### DISCUSSION THREAD

**Discussion by kukuro-kt** [actix/actix-web#3213] [General]
I wrote a macro to generate a base `HttpResponse`. I think others can modify something of it with `customize()`.

However, I found `customize().cookie(...)` is not supported.

Will actix support that? Or I have to manage cookie header manually?

**Comment by robjtede** [comment]
yea i think it's possible, could you open a feature request issue please

**Reply by kukuro-kt** [reply]
submitted: https://github.com/actix/actix-web/issues/3214
